<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QStatisticsWidget</class>
 <widget class="QWidget" name="QStatisticsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QStatisticsWidget</string>
  </property>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>300</y>
     <width>361</width>
     <height>150</height>
    </rect>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>660</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>操作运行</string>
   </property>
   <widget class="QPushButton" name="pushButton_runOnce">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>340</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>一键检测</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>运行状态</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>92</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>进度：</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_resultPrg">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>55</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>状态:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_resultOutput">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 48pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>NA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_resultStatus">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string>工位空闲中</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView_resultList">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>450</y>
     <width>361</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>140</y>
     <width>361</width>
     <height>150</height>
    </rect>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <widget class="QWidget" name="dockWidgetContents_5"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
